### PR TITLE
Fix issues with richtext tile (3.0.x)

### PR DESCRIPTION
**Pull request by frapell**
Related to issue #547

### CODE REVIEW

File: resources/js/mosaic.layout.js [hunk 1]
```diff
@@ -456,7 +456,8 @@ export default class LayoutManager {
                 // Check if outside toolbar
                 if ($(elm).parents(".mosaic-toolbar").length === 0) {
                     // Deselect tiles
-                    self.mosaic.document.querySelectorAll(".mosaic-selected-tile").forEach(function(el) {
+                    self.mosaic.document.querySelectorAll(".mosaic-selected-tile:not(.mosaic-tile-loading)").forEach(function(el) {
+                        $(el).data("mosaic-tile").blur();
                         el.classList.remove("mosaic-selected-tile");
```
Comment (petschki) [inline resources/js/mosaic.layout.js:461]: using the `blur()` method one line above should already remove the class.

File: resources/js/mosaic.layout.js [hunk 2]
```diff
@@ -1259,7 +1260,10 @@ export default class LayoutManager {
             // Select new tile and make it draggables
             if (new_tile && original_tile.length > 0) {
                 original_tile.mosaicAddDrag();
-                original_tile.data("mosaic-tile").initializeContent();
+                var tiletype = original_tile.data("mosaic-tile").getType();
+                if (tiletype === "plone.app.standardtiles.html") {
+                    original_tile.data("mosaic-tile").setupWysiwyg();
+                }
```
Comment (petschki) [inline resources/js/mosaic.layout.js:1266]: I would move this to the `initializeContent()` method of the tile API. There are the type switches already.

File: resources/js/mosaic.toolbar.js [hunk 3]
```diff
@@ -121,7 +121,7 @@ class Toolbar {
         var self = this;
 
         var selected_tile = self.mosaic.document.querySelector(".mosaic-selected-tile");
-        var tiletype = selected_tile ? selected_tile["mosaic-tile"].getType() : null;
+        var tiletype = selected_tile ? $(selected_tile).data("mosaic-tile").getType() : null;
```
Comment (petschki) [inline resources/js/mosaic.toolbar.js:124]: It shouldn't make a difference if we use `tile["mosaic-tile"]` or `$tile.data("mosaic-tile")` (besides performance) ... like in patternslib both is initialized. If that makes a difference, there might be something else wrong.

**Review by petschki — CHANGES_REQUESTED** [review]
Somehow the `mousemove` offset is broken when inserting a new tile. The new blurred tile is far away from the mouse pointer.

**Comment by petschki** [comment]
Hm. I'm having problems with this ... I cannot create a column layout anymore.

**Comment by frapell** [comment]
No longer needed, the issue is fixed with #550

**Pull request closed without merging**